### ISSUE TITLE
Training and Validation EPE fluctuating a lot on training FlownetC

### ISSUE THREAD

**Issue by tabshaikh** [NVIDIA/flownet2-pytorch#159]
I am training FlownetC from scratch(on long schedule) using the same scripts of Nvidia on FlyingChairs and validating  on MPI sintel clean but the training and validation EPE is fluctuating a lot, till now only 180K iterations have happened, am I doing something wrong or will the validation EPE converge after some iterations. 

### average_batch_EPE
![Screenshot from 2019-06-21 17-47-44](https://user-images.githubusercontent.com/31198893/59921880-c250a080-944c-11e9-88f2-370a0b937614.png)
### average_batch_MultiScale-L1
![Screenshot from 2019-06-21 17-47-55](https://user-images.githubusercontent.com/31198893/59921909-d98f8e00-944c-11e9-938d-d6607e63f309.png)
### average_batch_EPE
![Screenshot from 2019-06-21 17-50-12](https://user-images.githubusercontent.com/31198893/59922031-2a06eb80-944d-11e9-907a-d68d9ced28d0.png)

### average_batch_MultiScale-L1
![Screenshot from 2019-06-21 17-51-44](https://user-images.githubusercontent.com/31198893/59922045-35f2ad80-944d-11e9-8a03-66d1d7dea4e5.png)

**Comment by tabshaikh** [comment]
@fitsumreda could you help me on this :)

**Comment by Queenyy** [comment]
@tabshaikh Hi, have you successfully trained FlowNet2C? my loss curve is also fluctuating while training. could you please share your experience :) thanks a lot.

**Comment by duduheihei** [comment]
> @tabshaikh Hi, have you successfully trained FlowNet2C? my loss curve is also fluctuating while training. could you please share your experience :) thanks a lot.

I can not train FlowNet2/FlowNet2C with FlyingChairs/Sintel as  training dataset, have you solved the problem?
